module-presentation/src/main/kotlin/site/yourevents/invitation/api/InvitationApi.kt

```diff
@@ -13,17 +16,25 @@ import site.yourevents.principal.AuthDetails
 import site.yourevents.response.ApiResponse
 import java.util.UUID
 
+@RequestMapping("/invitation")
 interface InvitationApi {
     @Operation(summary = "초대장 생성")
-    @PostMapping("/invitation")
+    @PostMapping
     fun createInvitation(
         @RequestBody createInvitationRequest: CreateInvitationRequest,
         @AuthenticationPrincipal authDetails: AuthDetails,
     ): ApiResponse<CreateInvitationResponse>
 
     @Operation(summary = "초대장 QR 코드 조회")
-    @GetMapping("/invitation/qr")
+    @GetMapping("/qr")
     fun getQrCode(
         @RequestParam invitationId: UUID,
     ): ApiResponse<InvitationQrResponse>
+
+    @Operation(summary = "초대장 삭제")
+    @PatchMapping("/delete/{invitationId}")
```

Comment:
현재 MVP에 초대장 수정 기능이 없으므로, URL에서 `delete`라는 행위를 나타내는 동사를 제거하고, PathVariable만 사용하는 것이 좋아 보입니다!
```suggestion
    @PatchMapping("/{invitationId}")
```

Reply:
넵 알겠습니다!! 수정하였습니다.